protocols/identify/src/behaviour.rs

```diff
@@ -87,7 +87,13 @@ pub struct Config {
     /// The initial delay before the first identification request
     /// is sent to a remote on a newly established connection.
     ///
-    /// Defaults to 500ms.
+    /// Defaults to 0ms.
+    #[deprecated(
+        since = "0.52.0",
```

Comment:
```suggestion
        since = "0.42.1",
```

This needs to be the package version.